src/agent.cc

```diff
@@ -95,13 +93,9 @@ Agent::~Agent() {
 
 std::string Agent::str() {
   std::stringstream ss;
-  ss << kind_ << "_" << prototype_
-     << " ( "
-     << "ID=" << id_
-     << ", implementation=" << spec_
-     << ",  name=" << prototype_
-     << ",  parentID=" << parent_id_
-     << " ) ";
+  ss << kind_ << "_" << prototype_ << " ( "
+     << "ID=" << id_ << ", implementation=" << spec_ << ",  name=" << prototype_
+     << ",  parentID=" << parent_id_ << " ) ";
```

Comment:
Is this a strict interpretation of the style guide, or more of a "preference" that the autoformatter has?
I personally prefer the old style, but would not want to impose it as part of a style guide.  It is immensely logical and easier to read as a human.